compiler/src/yul/runtime/functions.rs

```diff
@@ -7,9 +7,11 @@ pub fn std() -> Vec<yul::Statement> {
         alloc(),
         alloc_mstoren(),
         free(),
-        ccopy(),
-        mcopy(),
-        scopy(),
+        calldata_to_mem(),
+        mem_to_sto(),
```

Comment:
can't we have naming like this `mcopys()`, `scopym()`, `mcopym()`, `scopys()`. It is more aligned with other runtime functions we have. We can think of a proper naming scheme that can be `location from + operation + location to`

Reply:
I think this looks alot better. thanks for the suggestion